include/fem-dem/ib_particles_dem.h

```diff
@@ -0,0 +1,182 @@
+//
+// Created by lucka on 2021-10-11.
+//
+
+#ifndef LETHE_IB_PARTICLES_DEM_H
+#define LETHE_IB_PARTICLES_DEM_H
+
+#include <core/ib_particle.h>
+#include <core/ib_stencil.h>
+#include <core/lethegridtools.h>
+
+#include <solvers/navier_stokes_base.h>
+
+#include <deal.II/base/tensor.h>
+
+#include <deal.II/dofs/dof_handler.h>
+#include <deal.II/dofs/dof_tools.h>
+
+using namespace dealii;
+using namespace std;
```

Comment:
We try to avoid using namespace std as to not confuse deal.II and std functions.
```suggestion
```